src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+* {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+.big_search {
+  position: static;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.bigSearch {
+  position: relative;
+  margin: 20px 0;
+}
+
+.bigImg {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.smallSearch {
+  position: relative;
+}
+
+.small_search {

```

Comment:
😕 what is the difference
keep your code clean and do not store such similar names

Reply:
Done)